source/topologies/rs-to-sharded.txt

```diff
@@ -0,0 +1,101 @@
+.. _c2c-rs-to-sharded:
+
+=======================================
+Sync a Replica Set to a Sharded Cluster
+=======================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+This page outlines the procedure to sync data between a replica set and a 
+sharded cluster using :program:`mongosync`. To perform a sync between two
+replica sets, see :ref:`c2c-quickstart`.
+
+Before You Begin
+----------------
+
+Before you begin your sync, define your source replica set, your 
+destination sharded cluster, and an administrative user that has permissions in 
+both clusters. You must also download ``mongosync``. To learn more, see 
```

Comment:
[suggestion] Since there are three things that a user needs to define, I would suggest putting those steps in a [list ](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#create-short-paragraphs)for better readability